content/01.main_content.md

```diff
@@ -24,10 +26,17 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in order to identify disparities in ISCB honorees, Le et al. [@doi:10.1016/j.cels.2021.07.007] used scientists' publication records as the expected honoree rate.
```

Comment:
"expected honoree rate" is a weird phrasing. Maybe "used scientists' publication records to approximate the pool of potential honorees"?

Reply:
hm... yes I can see it as a bit awkward.
What about:
"  For example, in order to identify disparities in ISCB honorees, Le et al. used scientists' publication records to approximate the expected diversity of potential honorees."

https://github.com/greenelab/disparities_opinion/pull/16/commits/018f34f2d05cc5c6e42e9c2a3cad0b53783973a2